inspirehep/submissions/views.py

```diff
@@ -165,6 +166,23 @@ def start_workflow_for_submission(self, submission_data, control_number=None):
         abort(503)
 
 
+@shared_task(ingore_result=False, max_retries=5)
+def async_create_ticket_with_template(
+    queue, requestor, template_path, template_context, title, recid=None
+):
+    try:
+        ticket = create_ticket_with_template(
+            queue, requestor, template_path, template_context, title, recid
+        )
+        if ticket == -1:
+            logger.error(
```

Comment:
isn't it better to raise an exception here? the celery task will then show as failed, and we will get a nice traceback on sentry. For a view it makes sense not to have a 500, but letting celery tasks blow up is probably OK.

Reply:
I think if we get a job and RT fails to send a ticket, it should fail as loud as possible. Since we receive notifications of failed celery tasks even on our chat, I'd go for that too.

Reply:
@harunurhan :point_up: